Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -764,21 +813,135 @@ protected PaletteBase()
         /// </summary>
         /// <param name="style">Style of button spec.</param>
         /// <returns>Color value.</returns>
-        public abstract Color GetButtonSpecImageTransparentColor(PaletteButtonSpecStyle style);
+        public virtual Color GetButtonSpecImageTransparentColor(PaletteButtonSpecStyle style)
+        {
+            switch (style)
+            {
+                case PaletteButtonSpecStyle.Generic:
+                    return Color.Empty;
+                case PaletteButtonSpecStyle.Close:
+                case PaletteButtonSpecStyle.Context:
+                case PaletteButtonSpecStyle.Next:
+                case PaletteButtonSpecStyle.Previous:
+                case PaletteButtonSpecStyle.ArrowLeft:
+                case PaletteButtonSpecStyle.ArrowRight:
+                case PaletteButtonSpecStyle.ArrowUp:
+                case PaletteButtonSpecStyle.ArrowDown:
+                case PaletteButtonSpecStyle.DropDown:
+                case PaletteButtonSpecStyle.PinVertical:
+                case PaletteButtonSpecStyle.PinHorizontal:
+                case PaletteButtonSpecStyle.FormClose:
+                case PaletteButtonSpecStyle.FormMin:
+                case PaletteButtonSpecStyle.FormMax:
+                case PaletteButtonSpecStyle.FormRestore:
+                case PaletteButtonSpecStyle.FormHelp:
+                case PaletteButtonSpecStyle.PendantClose:
+                case PaletteButtonSpecStyle.PendantMin:
+                case PaletteButtonSpecStyle.PendantRestore:
+                case PaletteButtonSpecStyle.WorkspaceMaximize:
+                case PaletteButtonSpecStyle.WorkspaceRestore:
+                case PaletteButtonSpecStyle.RibbonMinimize:
+                case PaletteButtonSpecStyle.RibbonExpand:
+                    return Color.Magenta;
+                case PaletteButtonSpecStyle.New | PaletteButtonSpecStyle.Open | PaletteButtonSpecStyle.SaveAll | PaletteButtonSpecStyle.SaveAs | PaletteButtonSpecStyle.Save | PaletteButtonSpecStyle.Cut | PaletteButtonSpecStyle.Copy | PaletteButtonSpecStyle.Paste | PaletteButtonSpecStyle.Undo | PaletteButtonSpecStyle.Redo | PaletteButtonSpecStyle.PageSetup | PaletteButtonSpecStyle.PrintPreview | PaletteButtonSpecStyle.Print | PaletteButtonSpecStyle.QuickPrint:
```

Comment:
This is generating a big flag bit value to match.
Please use the same style as above (i.e. each is explicitly stated)